packages/typespec-client-generator-core/lib/decorators.tsp

```diff
@@ -489,7 +494,7 @@ model ClientInitializationOptions {
   parameters?: Model;
 
   /**
-   * Determines how the client could be initialized. Use `InitializedBy` enum to set the value. The value could be `InitializedBy.individually`, `InitializedBy.parent` or `InitializedBy.individually | InitializedBy.parent`.
+   * Determines how the client could be initialized. Use `InitializedBy` enum to set the value. The value could be `InitializedBy.individually`, `InitializedBy.parent`, `InitializedBy.none` or `InitializedBy.individually | InitializedBy.parent`.
```

Comment:
Do we expect user write `InitializedBy.none`?

Or it is just TCGC output that means user didn't config it?

Reply:
user could not write `InitializedBy.none | InitializedBy.parent` which will cause diagnostic error.